hibernate-types-52/src/test/java/com/vladmihalcea/hibernate/util/providers/H2DataSourceProvider.java

```diff
@@ -19,41 +17,12 @@ public String hibernateDialect() {
     @Override
     public DataSource dataSource() {
         return JdbcConnectionPool.create(
-            url(),
```

Comment:
It's better to keep the existing setting as some of the existing committers might use them.